includes/wf_crm_webform_postprocess.inc

```diff
@@ -2414,7 +2414,7 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
           $val = array_search($val, $options);
         }
         // Fudge together date and time fields
-        if ($field['type'] === 'time' && substr($name, -8) === 'timepart') {
+        if (($field['type'] === 'time' || $field['type'] === 'webform_time') && substr($name, -8) === 'timepart') {
```

Comment:
Fixing the glue -> in D8 some elements are of type `webform_time` -> so make sure they also are found when date and time fields are glued together!